b2g/components/RemoteControlService.jsm

```diff
@@ -459,6 +454,28 @@ this.RemoteControlService = {
   _getIsCursorMode: function() {
     return RemoteControlService._isCursorMode;
   },
+
+  _getPairedDevices: function() {
+    let pairedDevices;
+    try {
+      pairedDevices = JSON.parse(Services.prefs.getCharPref(REMOTECONTROL_PREF_DEVICES));
+    } catch(e) {
+      DEBUG && debug ("Parse stored devices error: " + e);
+
+      // Empty paired devices
+      pairedDevices = {};
+      this._flushPairedDevices(pairedDevices);
+    }
+
+    return pairedDevices;
+  },
+
+  _flushPairedDevices: function(aPairedDevices) {
+    let data = JSON.stringify(aPairedDevices);
```

Comment:
If I use ```this._pairedDevices``` instead of ```aPairedDevices```, then ```generateUUID``` will be as follow:

```
generateUUID: function(aAES256Key, aHMAC256Key) {
    let uuidString = UUIDGenerator.generateUUID().toString();

    this._pairedDevices[uuidString] = {
      aesKey: aAES256Key,
      hmacKey: aHMAC256Key,
    };

    // Save this._pairedDevices so we can resume it for next start-up
    this._flushPairedDevices();

    return uuidString;
  },
```

The operations of ```this._pairedDevices``` will be hidden behind of the ```this._flushPairedDevices()```. I am not sure if it's ok, so I use ```aPairedDevices``` instead of ```this._flushPairedDevices()```. Could you give me some suggestions about this?